app/controllers/api/v1/slides_controller.rb

```diff
@@ -1,21 +1,21 @@
 class Api::V1::SlidesController < ApplicationController
-  before_action :set_slide, only: %i[show update destroy]
   before_action :authorize_request
+  before_action :set_slide, only: [:show, :update, :destroy]
   
   def index
     if admin?(@current_user)
       @slides = Slide.all
       render json: SlidesSerializer.new(@slides).serializable_hash.to_json
     else
-      render json: {msg: 'You are not authorized to perform that action'}, status: :forbidden
+      render json: {msg: 'You are not authorized to perform that action'}, status: :unauthorized
```

Comment:
Revisar el espaciado de las llaves en este archivo:
```suggestion
      render json: { msg: 'You are not authorized to perform that action' }, status: :unauthorized
```